docs/src/catalyst-standards/draft-cips/x509-envelope-metadata/cip-xxxx.md

```diff
@@ -0,0 +1,229 @@
+---
+CIP: /?
+Title: x509 Replayability Protection Envelope
+Category: MetaData
+Status: Proposed
+Authors:
+    - Steven Johnson<steven.johnson@iohk.io>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/?
+Created: 2023-10-24
+License: CC-BY-4.0
+---
+
+## Abstract
+
+x509 Certificate metadata and related metadata within an x509 Registration/Update transaction must be
+protected against replayability.  
+
+## Motivation: why is this CIP necessary?
+
+x509 certificate registration necessarily references data within the transaction body that it is attached to.
+Preventing replayability of the verbatim Auxiliary data attached to a transaction prevents the relationships
+between those references being altered and closes a potential attack vector where an unrelated registration update
+is attached to a different transaction.
+
+Rather than define an explicit issue with potential replayability of Metadata, we simply seek to prevent it
+to close off a whole category of potential attacks.
+
+## Specification
+
+The [x509 Metadata Envelope CDDL] defines the technical specification of the Metadata Envelope.
+The following description provides more information and context to that formal specification.
+Where this description and that specification are in disagreement, the specification prevails.
+
+The metadata envelope has the following basic structure:
+
+![x509 Secure Metadata Envelope structure](images/metadata-envelope-structure.svg)
+
+The purpose of this data is to lock a particular set of Auxiliary data verifiably to its original transaction.
```

Comment:
Maybe adding some details here would give more context to readers before the explanation made below.
Something like
```suggestion
The purpose of this data is to lock a particular set of Auxiliary data verifiably to its original transaction by using transactions input hash.
```